src/screens/Embed/ChartsTab.js

```diff
@@ -6,17 +6,16 @@ import { EmbedChartContainer } from './Embed.style';
 
 export default function ChartsTab({
   state,
```

Comment:
since state is no longer used this can be removed as well

Reply:
woops, good catch!